src/core/mac/mac_frame.cpp

```diff
@@ -254,63 +266,132 @@ Error Frame::ValidatePsdu(void) const
     return error;
 }
 
+#if OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE || OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
+bool Frame::IsWakeupFrame(void) const
+{
+    const uint16_t fcf    = GetFrameControlField();
+    bool           result = false;
+    uint8_t        keyIdMode;
+    uint8_t        firstIeIndex;
+    Address        srcAddress;
+
+    // Wake-up frame is a Multipurpose frame without Ack Request...
+    VerifyOrExit((fcf & kFcfFrameTypeMask) == kTypeMultipurpose);
+    VerifyOrExit((fcf & kMpFcfAckRequest) == 0);
+
+    // with extended source address...
+    SuccessOrExit(GetSrcAddr(srcAddress));
+    VerifyOrExit(srcAddress.IsExtended());
+
+    // Key Id Mode 2 secured...
```

Comment:
What does `...` mean here?

Reply:
That this comment is a part of sentence that is completed below :)

Reply:
Maybe it would be more clear if we used `...` also at the beginning of the subsequent comments after the first one?